load-testing/tests/relays_stress_helpers_test.go

```diff
@@ -1479,3 +1570,36 @@ func (s *relaysSuite) populateWithKnownGateways() (gateways []*accountInfo) {
 
 	return gateways
 }
+
+// WaitAll waits for all the provided functions to complete.
+// It is used to wait for multiple goroutines to complete before proceeding.
+func (s *relaysSuite) WaitAll(waitFuncs ...func()) {
```

Comment:
I noticed this doesn't use `s` at all. 

Can we move this into TestUtil?

Reply:
Good catch!